test/hotspot/jtreg/runtime/invokedynamic/BootstrapMethodErrorTest.java

```diff
@@ -43,23 +42,27 @@
 import java.lang.reflect.InvocationTargetException;
 import java.util.List;
 
+import static java.lang.classfile.ClassFile.*;
+import static java.lang.constant.ConstantDescs.*;
+
 public class BootstrapMethodErrorTest {
 
-    static abstract class IndyClassloader extends ClassLoader implements Opcodes {
+    static abstract class IndyClassloader extends ClassLoader{
 
         public IndyClassloader() {
             super(BootstrapMethodErrorTest.class.getClassLoader());
         }
 
         @Override
         public Class findClass(String name) throws ClassNotFoundException {
-            byte[] b;
+            byte[] b = new byte[0];
             try {
                 b = loadClassData(name);
             }
             catch (Throwable th) {
-                throw new ClassNotFoundException("Loading error", th);
+                th.printStackTrace();
```

Comment:
Why? printing stack trace is not recommended as it may lead to the throwable being ignored. Please restore the previous code here.

Reply:
did it for debugging and forgot about it, Thank you